lib/walk_nodes.js

```diff
@@ -148,14 +148,35 @@ module.exports = function walkNodes (AST) {
         } : null))
 
         if (declaration.init && declaration.init.properties)
-          declaration.init.properties.forEach(property =>
-            nodes.push(Object.assign(pickLocation(property), {
-              name: property.key.name,
-              type: fnTypes.has(property.value.type) ?
-                names.method : names.property
-            }, declaration.id ? {
-              target: declaration.id.name
-            } : null)))
+          declaration.init.properties
+            .map(property => {
+              if (property.type === 'SpreadElement')
+                return Object.assign(
+                  property,
```

Comment:
careful, this is mutating the original `property`, are you sure?

Reply:
Wow! Thanks for catching this!